libbeat/processors/dissect/field.go

```diff
@@ -186,7 +267,7 @@ type appendField struct {
 func (f appendField) Apply(b string, m Map) {
 	v, ok := m[f.Key()]
 	if ok {
-		m[f.Key()] = v + f.JoinString() + b
+		m[f.Key()] = v.(string) + f.JoinString() + b
```

Comment:
Also here we should do something safer to avoid panics if `v` is not a string.

Reply:
Applied same logic as above.